mlflow/h2o.py

```diff
@@ -108,7 +108,10 @@ def save_model(
         _save_example(mlflow_model, input_example, path)
 
     # Save h2o-model
-    h2o_save_location = h2o.save_model(model=h2o_model, path=model_data_path, force=True)
+    if "download_model" in dir(h2o):
```

Comment:
Why the if statement? Is there an h2o version that does not have download_model? I think we should just call download_model without the if statement.